semantics/language/execution/error.k

```diff
@@ -89,11 +89,11 @@ module C-EXECUTION-ERROR
           requires T =/=K void
 
      rule [err00037]:
-          (.K => UNDEF("CEER1", "Trying to read through a null pointer.", "6.3.2.1:1") ) ~>
+          (.K => UNDEF("CEER1", "Trying to read through a null pointer.", "6.3.2.1:1, J.2:1 item 19") ) ~>
           read(NullPointer, _)
           [structural]
      rule [err00046]:
-          (.K => UNSPEC("CEER2", "Trying to reinterpret integer bytes as floating bytes.", "6.2.6.1:1") ) ~>
+          (.K => UNSPEC("CEER2", "Trying to reinterpret integer bytes as floating bytes.", "6.2.6.1:5") ) ~>
```

Comment:
this was correct before. Paragraph 5 deals with trap representations which is not what this is; this is just talking about the unspecified representation of floating point types.